web/src/components/storage/ProposalActionsDialog.jsx

```diff
@@ -29,17 +29,19 @@ const ActionsList = ({ actions }) => {
   // Some actions (e.g., deleting a LV) are reported as several actions joined by a line break
   const actionItems = (action, id) => {
     return action.text.split("\n").map((text, index) => {
+      const Wrapper = action.delete ? "strong" : "span";
```

Comment:
Here I'd like to hear opinion from _storage people_. @ancorgs, to mention one.

Please, take your time to check https://html.spec.whatwg.org/#the-b-element and  https://html.spec.whatwg.org/#the-strong-element. What element do you think suits better for delete actions. In my view `strong` is the right one if my understanding about their importance is accurate.

Reply:
Yes. Looks like `strong` is the right thing.